src/fhir/input/fsh/profiles/HealthcareServiceDirectory.fsh

```diff
@@ -14,8 +14,12 @@ organisations specific for german Healthcare and Telematics Infrastructure.
 * meta.tag[Origin].system = "https://gematik.de/fhir/directory/CodeSystem/Origin"
 * id ..1 MS
 * providedBy 1..1 MS
+* category MS 
+* category from IHEXDShealthcareFacilityTypeCodePatientRelatedHealthcare
+* type MS 
+* type from HealthcareServiceTypeVS (extensible)
```

Comment:
HealthcareService.type wollen wir auch für den Typ der Organisation nutzen (https://simplifier.net/vzd-fhir-directory/organizationtypevs), wenn der HealthcareService im LDAP VZD eine Organisation ist. 
Hintergrund: Im LDAP VZD sind alle SMC-B TelematikIDs eine Organisation. Einige Einrichtungen haben sich mehrere SMC-Bs mit unterschiedlichen TelematikIDs geben lassen. Dafür wollen wir die Möglichkeit schaffen diese TelematikIDs bei der Synchronisierung in den FHIR VZD zu einer FHIR Organization zusammenzufassen. Den LDAP Organization.type der untergeordneten Organisation wollen wir im FHIR VZD in HealthcareService.type schreiben.
   * type MS
   * type from OrganizationTypeVS